frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -63,7 +65,13 @@ const VMConsoles: React.FC<VMConsolesProps> = ({
   serial,
   rdp,
   LoadingComponent,
```

Comment:
we could also add the url console type param for `/k8s/ns/default/virtualmachines/vm-example/consoles`